File: chap4/DivisionDecimale.py
# Ces programmes sont sous licence CeCILL-B V1.
def divisionDecimale (dividende, diviseur):
   if diviseur == 0:
      quotient = float("inf")
   else:
      quotient = dividende / diviseur
   return quotient

print( divisionDecimale(2,3) )

File: chap4/DivisionDecimale.js
// Ces programmes sont sous licence CeCILL-B V1.
function divisionDecimale (dividende, diviseur) {
	var quotient;
	//le test "diviseur nul" n'est pas utile (renvoie "Infinity")
	quotient = dividende / diviseur;//inutile de passer de int à float
	return quotient;
}
ISNprintln(divisionDecimale(2,3));
